src/gbtrf.cc

```diff
@@ -243,76 +249,84 @@ void gbtrf(
 ///
 /// @param[in] opts
 ///     Additional options, as map of name = value pairs. Possible options:
+///
 ///     - Option::Lookahead:
 ///       Number of panels to overlap with matrix updates.
 ///       lookahead >= 0. Default 1.
+///
 ///     - Option::InnerBlocking:
 ///       Inner blocking to use for panel. Default 16.
+///
 ///     - Option::MaxPanelThreads:
 ///       Number of threads to use for panel. Default omp_get_max_threads()/2.
+///
 ///     - Option::Target:
 ///       Implementation to target. Possible values:
 ///       - HostTask:  OpenMP tasks on CPU host [default].
 ///       - HostNest:  nested OpenMP parallel for loop on CPU host.
 ///       - HostBatch: batched BLAS on CPU host.
 ///       - Devices:   batched BLAS on GPU device.
 ///
-/// TODO: return value
+///    - Option::PivotThreshold:
+///      Strictness of the pivot selection.  Between 0 and 1 with 1 giving
+///      partial pivoting and 0 giving no pivoting.  Default 1.
+///
 /// @retval 0 successful exit
-/// @retval >0 for return value = $i$, $U(i,i)$ is exactly zero. The
-///         factorization has been completed, but the factor $U$ is exactly
+/// @retval i > 0, $U(i,i)$ is exactly zero, where i is a 1-based index.
+///         The factorization has been completed, but the factor $U$ is exactly
 ///         singular, and division by zero will occur if it is used
 ///         to solve a system of equations.
 ///
 /// @ingroup gbsv_computational
 ///
 template <typename scalar_t>
-void gbtrf(
+int64_t gbtrf(
     BandMatrix<scalar_t>& A, Pivots& pivots,
     Options const& opts )
 {
     Target target = get_option( opts, Option::Target, Target::HostTask );
+    int64_t info = 0;
 
     switch (target) {
         case Target::Host:
         case Target::HostTask:
-            impl::gbtrf<Target::HostTask>( A, pivots, opts );
+            info = impl::gbtrf<Target::HostTask>( A, pivots, opts );
```

Comment:
How would it cut the function length? It would save declaring `int64_t info = 0;` above, but we probably still need some `return foo;` below or the compiler will complain about reaching the end of a non-void function.

But I'm not opposed to that change.

Reply:
Remove all the break statements cuts the line count.  And, I misunderstood the rules on `enum class` types, so we'd need to add a `default` case to avoid the warning.  (Also, I did bad math, it'd reduce by more like a quarter.)

Reply:
Done.

Reply:
Done.